src/styles.css

```
@@ -23,6 +23,42 @@
+  --shadow-lg:0 12px 40px rgba(58,32,16,.15);
+  --card-bg:#FFFBF0;--nav-bg:rgba(245,238,220,.93);--toggle-bg:#DCCCAB;--toggle-knob:#FFFBF0;
+  --input-bg:#F5EEDC;--tag-bg:#F5EEDC;--modal-bg:#FFFBF0;--an-bg:#F5EEDC;
+  --skeleton-base:#e5d6bb;--skeleton-shine:#f1e6d4;
+}
+[data-theme="nord"]{
+  --bg:#2E3440;--white:#3B4252;
+  --orange:#88C0D0;--orange-deep:#8FBCBB;--orange-pale:#2E3440;--orange-mid:#3B4252;--orange-border:#4C566A;
+  --ink:#ECEFF4;--ink2:#E5E9F0;--ink3:#D8DEE9;--muted:#D8DEE9;
+  --border:#434C5E;--border2:#4C566A;
+  --green:#A3BE8C;--red:#BF616A;--blue:#81A1C1;--purple:#B48EAD;
+  --shadow:0 1px 4px rgba(0,0,0,.3),0 6px 20px rgba(0,0,0,.2);
```

Stylelint (17.12.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)